pubspec.yaml

```diff
@@ -13,6 +13,7 @@ environment:
 
 dependencies:
   args: ^2.0.0
+  kroki: ^0.0.2
```

Comment:
We almost certainly can't add a dependency to `kroki`.

Reply:
lol - yes that is only for the illustrative example - none of the example directory needs to be merged and this dependency would be removed.